redaxo/src/addons/structure/plugins/version/lang/es_es.lang

```diff
@@ -4,9 +4,12 @@ version_liveversion = Versión en vivo
 version_workingversion = Versión de trabajo
 version_working_to_live = Hacer una versión en vivo
 version_preview = Avance
+version_empty_workingversion = Vacía a versión de trabajo
 version_copy_live_to_workingversion = Copiar a versión de trabajo
 version_copy_from_liveversion = Obtener versión en vivo
 version_confirm_copy_live_to_workingversion = [Missing_text] de: Liveversion en Arbeitsversion kopieren? Alle bestehenden Inhalte der Arbeitsversion werden überschrieben.
 version_info_working_version_to_live = La versión de trabajo ya está en vivo
 version_info_live_version_to_working = La versión en vivo ha sido copiada a la versión de trabajo
 version_warning_working_version_to_live = La versión de trabajo no se puede copiar porque está vacía
+version_info_empty_workingversion = [Missing_text] de: Arbeitsversion erfolgreich geleert
```

Comment:
"[Missing_text]" bitte nicht verwenden, das hatten wir mal genutzt, aber seit traduko lieber leer/weg lassen (siehe oben).
Die noch vorhandenen "[Missing_text]" habe ich nun auch entfernt: https://github.com/redaxo/redaxo/pull/1863

Reply:
Ah ok, gut zu wissen für die Zukunft.
Ich hatte das in einer Datei gesehen und fand das eigentlich sinnvoll - aber das wird wohl dann jetzt script-seitig abgefangen, wenn die Übersetzung fehlt. Jetzt weiß ich es ja :) Thx!